api/seeds/test/roles.js

```diff
@@ -1,57 +1,52 @@
 exports.seed = async knex => {
-  // For testing, throw out the standard stuff and put in our
-  // hard-coded stuff so we know what IDs to expect.
-  await knex('auth_role_activity_mapping').del();
-  await knex('auth_roles').del();
-  await knex('auth_activities').del();
```

Comment:
Because role creation was automatic, the test seed previously had to delete them again so it could insert them with known IDs.  Now role creation is opt-in and the test seed doesn't opt-in, so it can just create the data it needs.